src/RecoveryServices/RecoveryServices.Backup.Models/AzureFileShareModels/AzureFileSharePolicy.cs

```diff
@@ -12,6 +12,8 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
+using System.Management.Automation;
```

Comment:
Please remove it if not necessary. I only see new namespace added without other changes.

Reply:
Removed